orm/src/main/java/org/hibernate/search/FullTextQuery.java

```diff
@@ -13,7 +13,6 @@
 import org.apache.lucene.search.Sort;
 
 import org.hibernate.Criteria;
-import org.hibernate.Query;
```

Comment:
It seems this import was being used in the javadocs, and the build now fails.
See lines 153 and 159.